plugins/modules/azure_rm_storageaccount.py

```diff
@@ -870,11 +981,12 @@ def account_obj_to_dict(self, account_obj, blob_mgmt_props=None, blob_client_pro
             allow_blob_public_access=account_obj.allow_blob_public_access,
             network_acls=account_obj.network_rule_set,
             is_hns_enabled=account_obj.is_hns_enabled if account_obj.is_hns_enabled else False,
+            allow_shared_key_access=account_obj.allow_shared_key_access if account_obj.allow_shared_key_access else False,
```

Comment:
It would be perfect if we could also increase this return value to azure_rm_storageaccount_info.py!